terraform/auth0/operations-engineering/clients.tf

```
@@ -194,7 +199,7 @@ resource "auth0_client" "operations_engineering_standards_report" {
   is_first_party                        = true
   is_token_endpoint_ip_header_trusted   = false
   logo_uri                              = null
-  name                                  = "Operations-Engineering-Standards-Report"
+  name                                  = "Standards Report"
```

```suggestion
  name                                  = "Repository Standards Report"
```

But then again the repo is called "operations-engineering-reports", but the website header has "MoJ Operations Engineering Reports" but then the Header says "Ministry of Justice GitHub Standards Report" 🤯